### DISCUSSION TITLE
Add the possibility, to add bookmarks from outside of daily.dev?

### DISCUSSION THREAD

**Discussion by tobiashochguertel** [dailydotdev/daily#954] [Feature Requests]
When a user, like me, doesn't have enough reputation he can't "Submit article", but I would like to at least track these articles which I want to read in my bookmarks of daily dev. 

Similar to my feature request:
- https://github.com/dailydotdev/daily/discussions/406
- https://github.com/dailydotdev/daily/discussions/629

**Comment by idoshamun** [comment]
Hi @tobiashochguertel,

Thanks for the suggestion. Let's keep the discussion on #629 :)
